gluten-substrait/src/main/scala/org/apache/spark/sql/execution/ui/GlutenUIUtils.scala

```diff
@@ -28,16 +28,13 @@ object GlutenUIUtils {
    * Check if enabled the gluten ui, Please note that, developer should pass the gluten config if
    * call from sql module to prevent misjudgment caused by users directly setting sparkConf.
    */
-  def uiEnabled(sc: SparkContext, glutenConfig: Option[GlutenConfig] = None): Boolean = {
-    val glutenTabEnabled = if (glutenConfig.isDefined) {
-      glutenConfig.get.glutenUiEnabled
-    } else {
-      sc.getConf.getBoolean(
-        GlutenConfig.GLUTEN_UI_ENABLED.key,
-        GlutenConfig.GLUTEN_UI_ENABLED.defaultValue.get)
-    }
-    sc.ui.isDefined && glutenTabEnabled
-  }
+  def uiEnabled(sc: SparkContext, glutenConfig: Option[GlutenConfig] = None): Boolean =
+    sc.ui.isDefined && glutenConfig
```

Comment:
Is it possible to keep only one parameter? i.e., `def uiEnabled(sc: SparkContext)`?

Reply:
I guess it will work. But it will lead to another consideration that is do we need glutenUiEnabled anymore?

Reply:
Done